bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -282,13 +292,31 @@
   public static final String BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY =
       "google.bigtable.mutate.rpc.timeout.ms";
 
+  /**
+   * If timeouts are set, how many milliseconds should pass before a DEADLINE_EXCEEDED for an RPC
+   * attempt within a long mutation. Note that multiple attempts may happen within an overall
+   * operation, whose timeout is governed by {@link #BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY}. Currently,
+   * this feature is experimental.
+   */
+  public static final String BIGTABLE_MUTATE_RPC_ATTEMPT_TIMEOUT_MS_KEY =
+      "google.bigtable.mutate.rpc.attempt_timeout.ms";
+
   /**
    * If timeouts are set, how many milliseconds should pass before a DEADLINE_EXCEEDED for a long
    * read. Currently, this feature is experimental.
    */
   public static final String BIGTABLE_READ_RPC_TIMEOUT_MS_KEY =
       "google.bigtable.read.rpc.timeout.ms";
 
+  /**
+   * If timeouts are set, how many milliseconds should pass before a DEADLINE_EXCEEDED for an RPC
+   * attempt within a long mutation. Note that multiple attempts may happen within an overall
+   * operation, whose timeout is governed by {@link #BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY}. Currently,
```

Comment:
Link here should be READ_RPC instead of MUTATE_RPC

Reply:
Fixed